packages/terra-time-input/CHANGELOG.md

```diff
@@ -19,6 +19,9 @@
 * Added
   * Added keyboard shortcuts `N`, `-`, `+`, `A`, and `P` for auto-setting time values.
 
+* Fixed
+  * Fixed some edge case scenarios for keyboard shortcuts
```

Comment:
This should be moved up to the Unreleased section.

Reply:
Fixed here. https://github.com/cerner/terra-framework/pull/1520/commits/e31d2250dc60b5bc379214c65e7312ff889cf584#diff-72cbeeaa7b7acb5b5ff08c157dd2db817a1cb487850786e696399735fb4898e1R5